airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/CatalogHelpers.java

```diff
@@ -41,46 +41,56 @@
 
 public class CatalogHelpers {
 
-  public static AirbyteCatalog createAirbyteCatalog(String streamName, Field... fields) {
-    return new AirbyteCatalog().withStreams(Lists.newArrayList(createAirbyteStream(streamName, fields)));
+  public static AirbyteCatalog createAirbyteCatalog(String namespace, String streamName, Field... fields) {
```

Comment:
See my comment above. We should always manipulate namespace/streamname as AirbyteStreamName, never as two separate entities